content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -33,13 +39,42 @@ resources:
       bucket: ((artifacts-bucket))
       versioned_file: ocw-hugo-themes/((ocw-hugo-themes-branch))/webpack.json
   # END ONLINE-ONLY
+
 # START ONLINE-ONLY
 task-config: &webhook-config
   platform: linux
   image_resource:
     type: registry-image
     source: {repository: curlimages/curl}
 # END ONLINE-ONLY
+
+task-purge-cdn-cache: &task-purge-cdn-cache
```

Comment:
Yaml anchor. See https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases, https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd, or your favorite google result